Resources/Prototypes/_NF/Catalog/Cargo/cargo_engineering.yml

```diff
@@ -0,0 +1,19 @@
+- type: cargoProduct
+  id: EngineeringRCDAmmo
+  icon:
+    sprite: Objects/Tools/rcd.rsi
+    state: ammo
+  product: CrateRCDAmmo
+  cost: 10000
+  category: Engineering
+  group: market
+
+- type: cargoProduct
+  id: EngineeringRCD
+  icon:
+    sprite: Objects/Tools/rcd.rsi
+    state: icon
+  product: CrateRCD
+  cost: 15000
+  category: Engineering
+  group: market
```

Comment:
No, this should never ever be in cargo, not to mention the crate was uncommented from the upstream file.

Reply:
It was removed from that ship.